tasks/e2e-brs.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
@jaysonwu991 Can you highlight what part of this script is different to the `e2e-simple.sh` one?

Reply:
`e2e-simple.sh` is to test the latest version of BRS, this makes `e2e-simple.sh` is useless for running, but for CI `e2e.brs.sh` tests the PR version directly, and it can directly address some issues through CI.

Reply:
The difference highlight here is that `e2e-simple.sh` and `e2e-installs.sh` are all about `Create React App`, it is nothing to do with `BRS` and `PR`, and the running also takes so much time to complete.
Even though there is no running anymore, these 2 scripts still can provide some ideas for us to write new scripts, so better to keep it as is and never run on GHA CI again.

Reply:
> Even though there is no running anymore, these 2 scripts still can provide some ideas for us to write new scripts, so better to keep it as is and never run on GHA CI again.

These old scripts will only be useful if there is context on what they do, and why they aren't being referenced. Otherwise we're leaving dead code around that will only confuse someone in the future.

If you want to keep them around can you write a README in the `/tasks` directory to explain each script please

Reply:
Hi @runmoore , have removed these 2 scripts, we can restore these 2 from Git if needed in the future.